package.json

```
@@ -9,7 +9,12 @@
   "repository": "github:deftomat/opinionated",
   "author": "Tomas Szabo <tomas.szabo@deftomat.com>",
   "homepage": "https://github.com/deftomat/opinionated",
-  "exports": "./dist/esm/index.js",
+  "exports": {
+    "./dist/esm/index.js": "./dist/esm/index.js",
+    "./configs/eslint.mjs": "./configs/eslint.mjs",
+    "./configs/prettier.config.mjs": "./configs/prettier.config.mjs",
+    "./configs/tsconfig.json": "./configs/tsconfig.json"
```

```suggestion
    "./": {
       "types": "./dist/esm/index.d.ts",
       "import": "./dist/esm/index.js"
    },
    "./configs/eslint.mjs": "./configs/eslint.mjs",
    "./configs/prettier.config.mjs": "./configs/prettier.config.mjs",
    "./configs/tsconfig.json": "./configs/tsconfig.json"
```

Also, `prepublishOnly` should be `rm -rf dist && npx tsc`